src/AzureDeprecation.APIs.REST/Utils/JsonSettingsProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+
+namespace AzureDeprecation.APIs.REST.Utils;
+
+public static class JsonSettingsProvider
+{
+    public static JsonSerializerSettings GetDefault()
```

Comment:
We already have a serializer that you can use. Can you use that one instead please?